marlowe-playground-client/src/Projects/View.purs

```diff
@@ -0,0 +1,104 @@
+module Projects.View (render) where
+
+import Prelude hiding (div)
+import Data.Array (filter)
+import Data.DateTime.ISO as ISO
+import Data.Either (Either(..))
+import Data.Formatter.DateTime (FormatterCommand(..), format)
+import Data.Lens (to, view, (^.))
+import Data.List (fromFoldable)
+import Data.Maybe (Maybe(..))
+import Data.Newtype (unwrap)
+import Effect.Aff.Class (class MonadAff)
+import Gist (Gist, gistCreatedAt, gistDescription, gistId, gistUpdatedAt)
+import Halogen (ClassName(..), ComponentHTML)
+import Halogen.Classes (fontSemibold, modalContent, paddingRight, smallPaddingRight, textSm)
+import Halogen.HTML (HTML, a, a_, div, div_, span, text)
+import Halogen.HTML.Events (onClick)
+import Halogen.HTML.Properties (class_, classes)
+import MainFrame.Types (ChildSlots)
+import Marlowe.Gists (fileExists, filenames, playgroundGist)
+import Modal.ViewHelpers (modalHeaderWithClose)
+import Network.RemoteData (RemoteData(..))
+import Prim.TypeError (class Warn, Text)
+import Projects.Types (Action(..), Lang(..), State, _projects, modalIsLoading)
+import Text.Parsing.Parser (runParser)
+
+render ::
+  forall m.
+  Warn (Text "Fix 'body Loading' design") =>
+  MonadAff m =>
+  State ->
+  ComponentHTML Action ChildSlots m
+render state =
+  if modalIsLoading state then
+    text mempty
+  else
+    div_
+      [ modalHeaderWithClose "Open Project" Cancel
+      , div [ classes [ modalContent ] ]
+          [ body playgroundGists
+          ]
+      ]
+  where
+  playgroundGists = filter playgroundGist <$> state ^. _projects
```

Comment:
Small thing, but we should rename this predicate to is `isPlaygroundGist`. This line would be easier to read.

Reply:
is not a predicate, is the list of gists

Reply:
`playgroundGists` (plural) is a list. But `playgroundGist` (singlar) is a predicate, no? I mean the singular version.

Reply:
agree with Kris, would need to rename `Marlowe.Gists.playgroundGist` to `Marlowe.Gists.isPlaygroundGist`

Reply:
done :)